### PR TITLE
Migrate ballerina-java.jdbc module to separate repository

### PR DESCRIPTION
**Pull request by niveathika**
## Purpose
$subject as part of task https://github.com/ballerina-platform/ballerina-lang/issues/25085

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Test environment
H2

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
-**Description:**
```
Comment (BuddhiWathsala) [inline .github/issue_template.md:1]: Any reason to move issue template inside `.github` directory?

**Review by BuddhiWathsala — CHANGES_REQUESTED** [review]

File: .github/issue_template.md (hunk 1 above)
Comment (niveathika) [reply .github/issue_template.md:1]: Thought it will be much cleaner, with multiple files at the root level, it seemed bit cluttered

Comment (BuddhiWathsala) [reply .github/issue_template.md:1]: Isn't that affects when we create issues?

Comment (niveathika) [reply .github/issue_template.md:1]: Yes the config looks for these under `.github` too. Check out https://github.com/ballerina-platform/module-ballerinax-mongodb/issues

**Review by BuddhiWathsala — APPROVED** [review]

**Pull request merged**